package.json

```diff
@@ -6,12 +6,13 @@
   "main": "service.js",
   "scripts": {
     "start": "node .",
-    "dev": "nodemon .",
```

Comment:
keep the `nodemon .` rather then the `nodemon service.js`
it should be the same as start

Reply:
interesting. I did a checkout of origin/master. How did i lose this line is surprising.